app/assets/stylesheets/components/_option-select.scss

```diff
@@ -176,16 +97,91 @@
   fill: currentColor;
 }
 
-.js-enabled .app-c-option-select__icon--up {
-  display: block;
+.app-c-option-select__container {
+  position: relative;
+  height: 200px;
+  background-color: $page-colour;
+  overflow-y: auto;
+  overflow-x: hidden;
+}
+
+.app-c-option-select__container-inner {
+  padding: $gutter-one-quarter;
 }
 
-.js-closed {
+.app-c-option-select__filter {
+  display: none;
+
+  .govuk-form-group {
+    margin-bottom: $gutter-one-third;
+  }
+}
+
+.js-enabled {
+  .app-c-option-select {
+    padding: 0 0 1px;
+  }
+
+  .js-selected-counter {
+    @include core-14;
+  }
```

Comment:
This class can be removed if you add `govuk-!-font-size-14` to https://github.com/alphagov/finder-frontend/blob/master/app/assets/javascripts/components/option-select.js#L141